### PR TITLE
Fix PHPMD violations

### PR DESCRIPTION
**Pull request by julienfalque**
I just ran PHPMD on the codebase and fixed reported issue. This will prevent PHPMD errors for the next PR that will change the file.

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
Why do we get phpmd to only run on the changed files? Just get it to check everything?

File: src/Fixer/FunctionNotation/VoidReturnFixer.php [hunk 1]
```diff
@@ -203,8 +203,6 @@ private function hasVoidReturn(Tokens $tokens, $startIndex, $endIndex)
     }
 
     /**
-     * Apply the fix to the function definition.
```
Comment (GrahamCampbell) [inline src/Fixer/FunctionNotation/VoidReturnFixer.php]: Instead of removing this, we could make it more informative?

**Comment by SpacePossum** [comment]
>Why do we get phpmd to only run on the changed files? Just get it to check everything?

performance, there is no need to run it on everything all the time. We run in this case because a PHPMD violation was merged to master, however the should no longer happen when 2.4 because the lowest maintained version (IIRC).

File: src/Fixer/FunctionNotation/VoidReturnFixer.php (hunk 1 above)
Comment (SpacePossum) [inline src/Fixer/FunctionNotation/VoidReturnFixer.php]: the method name says it all?

**Comment by GrahamCampbell** [comment]
> performance, there is no need to run it on everything all the time

Possible to implement a cache I guess. Sounds like a job for a different CI service, such as scrutinizer.

**Comment by SpacePossum** [comment]
could be done, but would take considerable time and now this little thingy is blocking PR's so maybe in another follow up PR?

File: src/Fixer/FunctionNotation/VoidReturnFixer.php (hunk 1 above)
Comment (GrahamCampbell) [inline src/Fixer/FunctionNotation/VoidReturnFixer.php]: The same is true for many of the other methods in this class, which phpmd has not picked up though too.

Comment (SpacePossum) [inline src/Fixer/FunctionNotation/VoidReturnFixer.php]: yeah, I would delete them all, but than I again I know I'm a bit of a minimalist on these things

Comment (julienfalque) [inline src/Fixer/FunctionNotation/VoidReturnFixer.php]: Indeed, but the purpose of this PR is mostly to
> prevent PHPMD errors for the next PR that will change the file.

**Comment by keradus** [comment]
Thank you @julienfalque.

**Pull request merged**